### PR TITLE
Add master to top of spec

### PR DESCRIPTION
**Pull request by mattmcneeney**
This reinforces that master branch is a work in progress.

### CODE REVIEW

**Comment by mattmcneeney** [comment]
New PR with the fix we discussed at the F2F
@duglin @avade @vaikas-google @pmorie

**Comment by angarg12** [comment]
LGTM

**Comment by duglin** [comment]
YUGE!
LGTM

**Comment by vaikas-google** [comment]
LGTM

File: spec.md [hunk 1]
```diff
@@ -1,4 +1,4 @@
-# Open Service Broker API
+# Open Service Broker API (master)
```
Comment (pmorie) [inline spec.md]: How do you feel about `master - may contain changes that are not yet released`

Comment (avade) [inline spec.md]: That is clearer to me. +1

Comment (mattmcneeney) [inline spec.md]: Nice idea, updated :)

**Comment by mattmcneeney** [comment]
Another round needed, thanks for the improvement @pmorie 

@angarg12 @vaikas-google @avade

**Comment by avade** [comment]
LGTM

**Comment by duglin** [comment]
LGTM

**Comment by angarg12** [comment]
lgtm

**Comment by pmorie** [comment]
LGTM

**Pull request merged**